modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/processor/ReconciliationResultCollector.java

```diff
@@ -101,12 +104,13 @@ public interface ReconciliationResultCollector {
      * @param partId Partition id.
      * @param conflicts Conflicted entries.
      * @param actualKeys Actual values.
+     * @throws IgniteException If cache with the given name does not exist.
```

Comment:
It seems a bit too generic to consider that IgniteException will be thrown only if cache with the given name does not exist, even if it's true for now. I'd rather create CacheNotFound extends IgniteException or similar. Up to you.

Reply:
It might be a good idea, but IMHO, creating a new exception class for the sake of one or two methods seems overkill.